jquery.sumoselect.js

```diff
@@ -772,6 +772,7 @@
 
         unload: function () {
           var O = this;
+          O.E.trigger('sumo:unloading', O);
```

Comment:
Could you give me an example where this event would be used?  
I understand the `unloaded` and `initialized`, but this one seems a bit too much

Reply:
Hi, this could be used when we want to show some progress to the end users when we are dealing with large lists. However, after reconsideration I've removed it since this will rarely be applicable. Thanks.